### ISSUE TITLE
Move pbt-gen under the tools folder

### ISSUE THREAD

**Issue by greg-szabo** [informalsystems/tendermint-rs#869]
pbt-gen seems a standalone tool. I think it should be under the tools folder. Can we move it?

**What's the definition of "done" for this issue?**
A: pbt-gen works and is moved under the "tools" folder.
or
B: A decision with a reason is documented to keep it where it is.

**Comment by thanethomson** [comment]
Maybe @shonfeder can comment on this in more detail, but I think it's in the root of the project because it's a support library used during testing (i.e. it's compiled into other crates), as opposed to a separate utility like the other tools.

**Comment by shonfeder** [comment]
Yeah, it's a library, not a standalone tool and it doesn't strike me as a "tool" in an obvious sense. Still, if it can be imported as a library from within the `tools` directory and there's some benefit to putting it there, I don't have an objection. But I also don't know what the current problem with the current placement is or what would be gained by moving it.
